transaction/core/test-utils/src/lib.rs

```diff
@@ -189,7 +189,7 @@ pub fn initialize_ledger<L: Ledger, R: RngCore + CryptoRng>(
                 let block_contents = BlockContents::new(key_images, outputs);
 
                 let block = Block::new(
-                    BLOCK_VERSION,
+                    0,
```

Comment:
In the tests, yes. I don't see a good alternative for this, since we use `BLOCK_VERSION` things will break once it moves to `2` since nothing will create a block with version `1` to bridge between `0` and `2`.

Why do you think it would break when we release v2?

Reply:
If we initialized a ledger at `0`, and tried to append a new block to it in the normal way when `BLOCK_VERSION >= 2`, it would blow up, no? This is `mc_consensus_enclave_impl::tests::form_block_refuses_duplicate_root_elements()`.

Reply:
This appears moot at this point.